setup.cfg

```diff
@@ -39,6 +39,9 @@ tests_require =
 [options.packages.find]
 where=python
 
+[options.package_data]
+lsst.pipe.base = py.typed
```

Comment:
Probably should be `lsst.ctrl.mpexec = py.typed`.... It can be fixed the next time someone works on the package.

Reply:
🤦‍♂️ Sorry, pasting without thinking. I'm fixing it already on my next ticket.